modoboa/core/drf_authentication.py

```diff
@@ -44,3 +45,8 @@ def authenticate(self, request):
         if user.tfa_enabled:
             self.verify_user(request, user)
         return user, token
+
+
+class SimpleJWTWith2FAScheme(SimpleJWTScheme):
```

Comment:
Where do you use this class?

Reply:
Nowhere, or at least not from the Modoboa side. From what I understood, it is dynamically added to the drf-speculator's extension registry when the class is defined - somewhere [here](https://github.com/tfranzel/drf-spectacular/blob/0.17.2/drf_spectacular/plumbing.py#L563).